packages/components/src/base-field/test/index.tsx

```diff
@@ -8,13 +8,17 @@ import { render } from '@testing-library/react';
  */
 import { useBaseField } from '../index';
 import { View } from '../../view';
+import type { WordPressComponentProps } from '../../ui/context';
+import type { UseBaseFieldProps } from '../types';
 
-const TestField = ( props ) => {
+const TestField = (
+	props: WordPressComponentProps< UseBaseFieldProps, 'div' >
+) => {
```

Comment:
`WordPressComponentProps` is not necessary here

```suggestion
const TestField = ( props: UseBaseFieldProps ) => {
```